samples/apps/bookinfo/bookinfo-ratings-v2.yaml

```diff
@@ -26,17 +26,25 @@ spec:
     spec:
       containers:
       - name: ratings
-        image: quay.io/saurabh/examples-bookinfo-ratings-v2:v1
+        image: quay.io/saurabh/examples-bookinfo-ratings-alldb:v1
         imagePullPolicy: IfNotPresent
         env:
-          - name: MYSQL_DB_HOST
-            value: mysqldb
-          - name: MYSQL_DB_PORT
-            value: "3306"
-          - name: MYSQL_DB_USER
-            value: root
-          - name: MYSQL_DB_PASSWORD
-            value: password
+          # ratings-v2 will use mongodb as the default db backend.
```

Comment:
which file?

Reply:
The file "bookinfo-mysql.yaml" is the reference to the mysql service which is still there.
The file bookinfo-ratings-v2.yaml can be modified to use the mysql service instead of mongodb.

Reply:
sorry I wasn't clear: I mean can we have 2 files rather than asking the user (or my test) to edit one of them

Reply:
done.